packages/application-config/schema.json

```diff
@@ -27,26 +23,11 @@
     },
     "entryPointUriPath": {
       "description": "See https://docs.commercetools.com/custom-applications/api-reference/application-config#entrypointuripath",
-      "oneOf": [
-        {
-          "type": "string",
-          "pattern": "^[^\\-_#]([0-9a-z]|[\\-_](?![\\-_])){0,62}[^\\-_#]$"
-        },
-        {
-          "$ref": "#/definitions/envVariablePlaceholder"
-        }
-      ]
+      "type": "string"
     },
     "cloudIdentifier": {
       "description": "See https://docs.commercetools.com/custom-applications/api-reference/application-config#cloudidentifier",
-      "oneOf": [
-        {
-          "enum": ["gcp-au", "gcp-eu", "gcp-us", "aws-fra", "aws-ohio"]
-        },
-        {
-          "$ref": "#/definitions/envVariablePlaceholder"
-        }
-      ]
```

Comment:
`entryPointUriPath` validation was extracted to the `validations` file but it seems we don't validate `cloudIdentifier` value anymore. Shouldn't we also extract this validation?

Reply:
Actually we already kind of validated it here, when we process the value: https://github.com/commercetools/merchant-center-application-kit/blob/66c2067cf9a30ce624fe8f98c9a616c185115498/packages/application-config/src/utils.ts#L5-L28

Reply:
Ah! Ok 👍 
It seems the comment is now outdated 😅